src/test/fuzz/policy_estimator_io.cpp

```diff
@@ -11,9 +12,14 @@
 #include <cstdint>
 #include <vector>
 
+namespace {
+const BasicTestingSetup* g_setup;
+} // namespace
+
 void initialize_policy_estimator_io()
 {
     static const auto testing_setup = MakeNoLogFileContext<>();
+    g_setup = testing_setup.get();
```

Comment:
Having a global dangling pointer here is kind of ugly. Is there really no other way to get this information in?
If you need a global, what about storing the path in the global instead of a pointer. It seems slightly safer at least.

Reply:
Yeah, this is the general pattern. Let me know if there is a better way.

And there should be no risk in doing "risky" stuff in the unit or fuzz tests, such as globals, raw pointers/memory, "wild" casts, etc..., as they are run in sanitizers that should detect any issue if it arises. (As opposed to real code, which is not 100% covered by sanitizers)

Reply:
Ok, that's a good point. I was mainly afraid of false positives/negatives due to pointers pointing to the wrong memory. But yes, the sanitizers should catch that.